zstacklib/zstacklib/utils/sanlock.py

```
@@ -178,6 +219,15 @@ def direct_init_resource(resource):
     cmd = "sanlock direct init -r %s" % resource
     return bash.bash_r(cmd)
 
+@bash.in_bash
+def direct_init_resource(vg_uuid, resource_uuid, offset):
```

Ruff (0.8.2)

Redefinition of unused `direct_init_resource` from line 218

(F811)